test/Microsoft.AspNet.Mvc.Common.Test/TaskHelperTest.cs

```diff
@@ -1,58 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Could you verify there were tests for the places that were using these methods (and if not add some to ensure we work correctly \ bubble the exception?)